js/script.js

```diff
@@ -0,0 +1,7 @@
+var a = prompt('Enter the triangle base'),
+    h = prompt('Enter the triangle height'),
+    triangleArea;
```

Comment:
wiem, że tak robi Kodilla ale nie deklaruj zmiennych po przecinkach - to zła praktyka. Każda zmienna to `var` od nowej linijki (i średnik po deklaracji)